frontEnd/src/views/Forgot.vue

```diff
@@ -1,19 +1,104 @@
 <template>
-  
-  <h1>This is the forgot password page</h1>
+<v-container fluid style="margin: 0 auto 0 auto; padding: 0px; max-width: 800px; width: 90% !important">
+  <v-card>
+    <v-img  
+      src="../assets/thumbnail_Calgary_horizontal_primary_CMYK_EN.png"
```

Comment:
This is becoming used so frequently that it should be turned into its own component